tests/baselines/reference/objectTypesIdentityWithConstructSignatures2.types

```diff
@@ -32,8 +32,8 @@ var a: { new(x: Date): string }
 >x : Date
 
 var b = { new(x: RegExp) { return ''; } }; // not a construct signature, function called new
->b : { new(x: RegExp): string; }
->{ new(x: RegExp) { return ''; } } : { new(x: RegExp): string; }
+>b : { "new"(x: RegExp): string; }

```

Comment:
This doesn't seem right... aren't all of these cases changed explicitly construct signatures that shouldn't be quoted?

Reply:
This one definitely isnt wrong. Notice how this is inferred~ from an object literal and that cant even have a construct signature - it’s just a method called `new`

Reply:
Agh, yeah. I think I skimmed this PR on my phone on the weekend and had it in my head that there was something going on.